af/fastpath/fastpath/core.py

```diff
@@ -96,7 +96,7 @@ def setup():
     ap.add_argument("--devel", action="store_true", help="Devel mode")
     ap.add_argument("--nossh", action="store_true", help="Do not start SSH feeder")
     ap.add_argument("--stdout", action="store_true", help="Log to stdout")
-    ap.add_argument("--db-uri", help="Database DSN or URI. The string is logged!")
+    ap.add_argument("--db-uri", help="Database DSN or URI.")
```

Comment:
Is the string not logged anymore?

Reply:
Correct: https://github.com/ooni/pipeline/pull/324/files#diff-74bb3253bea8a477a7d40a448b48f775L46